melodic/distribution.yaml

```diff
@@ -6,6 +6,14 @@ release_platforms:
   ubuntu:
   - bionic
 repositories:
+  ROS_driver-release:
```

Comment:
As a single package repository please make the key the same name as the package. This name makes some sense in the context of the hosting organization, but in the rosdistro context this name provides little to no information.

Reply:
Thanks for your answer. The name of the repo will be changed and the entries for doc and source will be added.  I will initiate a new release as soon as we know why the release was not successful (your 2nd comment)